backend/database_handler/migration/versions/a32f85df2806_add_client_session_id_to_transactions.py

```diff
@@ -0,0 +1,34 @@
+"""add client session id to transactions
+
+Revision ID: a32f85df2806
+Revises: 3566595124f6
+Create Date: 2024-10-01 12:09:19.995482
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "a32f85df2806"
+down_revision: Union[str, None] = "3566595124f6"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "transactions",
+        sa.Column("client_session_id", sa.String(length=255), nullable=True),
```

Comment:
@AgustinRamiroDiaz @denishacquin This works, but it doesn't comply with the transactions and the consensus we are simulating here. 
I would say that for now, it is ok if it works as expected, but we need to search for a solution at the WebSockets connection level instead of changing the transactions model.

Reply:
Fully agree.
This was the quickest solution. I think that the long term solution would be that the frontend should subscribe to the transactions that it wants to see, and the backend should broadcast to all subscribers